src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.